frontend/src/pages/accounts/settings.page.tsx

```diff
@@ -157,6 +164,43 @@ const Settings: NextPage = () => {
                     {labelCollectionWarning}
                   </div>
                 </div>
+                <div className={styles.field}>
+                  <h2 className={styles["field-heading"]}>
+                    {l10n.getString("setting-label-api-key")}
```

Comment:
question: we submitted this to the l10n repository right? I noticed this here: 
<img width="245" alt="image" src="https://user-images.githubusercontent.com/3924990/170792502-9321b9e4-74f3-401b-b2ec-0b06f618604f.png">

Reply:
I'm seeing it correctly; you might not be up-to-date with the l10n repo submodule yet? Here's a tip that might be interesting (I should submit a PR to the README with this): https://mozilla.slack.com/archives/C013CSYEL5T/p1646304489835969